### PR TITLE
LPS-145476 Modify methods to obtain the URL with the new FriendlyURL

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
https://issues.liferay.com/browse/LPS-145476

https://user-images.githubusercontent.com/47524751/165071743-60bf306f-aaf5-4c22-8515-8caa64e991c5.mp4

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Review by ambrinchaudhary — APPROVED** [review]

File: modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/helper/DLURLHelperImpl.java [hunk 1]
```diff
@@ -441,6 +435,50 @@ private String _getDLFileVersionURLProviderURL(
 		return null;
 	}
 
+	private String _getFriendlyURL(
+		boolean appendVersion, FileEntry fileEntry, String previewURLPrefix,
+		String queryString) {
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/helper/DLURLHelperImpl.java:440]: The parameter sorting looks a bit weird (lexicographic order). Why not follow the same "logical" ordering as in `_getPreviewUuidURL`?

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
@adolfopa I made the suggested changes and rebased due to branch conflicts

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**